grizzly-ls/tests/unit/test___main__.py

```diff
@@ -146,8 +179,81 @@ def test_setup_logging(mocker: MockerFixture, capsys: CaptureFixture[str]) -> No
     assert capture.out == ''
 
 
+def test_setup_debugging(mocker: MockerFixture, capsys: CaptureFixture[str]) -> None:
+    mocker.patch('grizzly_ls.__main__.logging.info')
+    logging_error_mock = mocker.patch('grizzly_ls.__main__.logging.error')
```

Comment:
use `_pytest.logging.LogCaptureFixture`, `with caplog.at_level(logging.ERROR)` and `caplog.messages == [..]` instead.